src/main/sh/run_bridge.sh

```diff
@@ -11,6 +11,6 @@ fi
 # configuring the bridge to work in "client" mode connecting to the messaging (router) layer
 export AMQP_MODE="CLIENT"
 export AMQP_HOST=$MESSAGING_SERVICE_HOST
-export AMQP_PORT=$MESSAGING_SERVICE_PORT_INTERNAL
+export AMQP_PORT=$MESSAGING_SERVICE_PORT_AMQPS_BROKER
```

Comment:
of course this change is needed and I'm ok with that but I have a comment on that. I'll open an issue involving you and Ulf as well to discuss about my proposal.